compiler/luci/pass/src/FusePreActivationBatchNormPass.cpp

```diff
@@ -114,11 +114,157 @@ bool is_batchnorm_add(const luci::CircleAdd *add, luci::CircleMul *&mul, luci::C
   return true;
 }
 
+luci::CircleConv2D *get_forward_conv2d(luci::CircleNode *node, uint32_t channel_size)
```

Comment:
```suggestion
luci::CircleConv2D *get_forward_conv2d(const luci::CircleNode *node, uint32_t channel_size)
```
This can be const as well.

Reply:
Updated. Thanks.